torch/nn/modules/module.py

```diff
@@ -378,6 +380,9 @@ def to(self, *args, **kwargs):
                 the floating point parameters and buffers in this module
             tensor (torch.Tensor): Tensor whose dtype and device are the desired
                 dtype and device for all parameters and buffers in this module
+            memory_format (:class:`torch.memory_format`): the desired memory
```

Comment:
are you planning to have a reference section for memory_format that you can point to?  This description isn't full enough for the long term.  (e.g. why only 4D parameters/buffers? -- it's not clear)

Reply:
Yes as soon as we land new defaults for .clone .to *_like ops I will work on updating docs.